src/main/java/com/julienvey/trello/domain/TrelloEntity.java

```diff
@@ -10,7 +10,16 @@
     @JsonIgnore
     protected Trello trelloService;
 
-    public void setInternalTrello(Trello trelloService) {
+    @SuppressWarnings("unchecked")
+    public <T extends TrelloEntity> T setInternalTrello(Trello trelloService) {
         this.trelloService = trelloService;
+        return (T) this;
+    }
+
+    protected Trello getTrelloService() {
```

Comment:
Allows users to get more informative exception, exactly describing problem and hinting how to solve it.